src/Admin/Extensions/Extensions.php

```diff
@@ -355,6 +329,45 @@ static function ( $a, $b ) {
 			}
 		);
 
+		return $populated_extensions;
+	}
+
+	/**
+	 * Get the list of WPGraphQL extensions.
+	 *
+	 * @return PopulatedExtension[] The list of extensions.
+	 */
+	public function get_extensions(): array {
+		if ( ! isset( $this->extensions ) ) {
+			// @todo Replace with a call to the WPGraphQL server.
+			$extensions = Registry::get_extenions();
+
+			/**
+			 * Filter the list of extensions, allowing other plugins to add or remove extensions.
+			 *
+			 * @see Admin\Extensions\Registry::get_extensions() for the correct format of the extensions.
+			 *
+			 * @param array<string,mixed> $extensions The list of extensions.
+			 */
+			$extensions = apply_filters( 'graphql_get_extensions', $extensions );
+
+			$valid_extensions = [];
+			foreach ( $extensions as $extension ) {
+				$sanitized = $this->sanitize_extension( $extension );
```

Comment:
Doing this in a separate step from validation, to reduce the tech debt of an eventual migration to a server payload.

Reply:
> eventual migration to a server payload

What is this server payload?

Reply:
E.g. `wp_remote_get( 'https://wpgraphql.com/{wp-json-backend-proxy}/extensions' );`

If the payload of extensions is coming from outside the plugin, we need to sanitize/validate it. So I'm trying to follow conventions that would require minimal changes to that pattern switch.